```
$ make test-php-unit NOCOVERAGE=true TEST_PHP_SUITE=tests/lib/Files/External/StoragesBackendServiceTest.php
...
PHPUNIT="/home/phil/git/owncloud/core/lib/composer/phpunit/phpunit/phpunit" build/autotest.sh sqlite tests/lib/Files/External/StoragesBackendServiceTest.php
Using PHP executable /usr/bin/php
Parsing all files in lib/public for the presence of @since or @deprecated on each method...

Using database oc_autotest
Setup environment for sqlite testing on local storage ...
Updated 0 paths from the index
Installing ....
creating sqlite db
ownCloud was successfully installed
Testing with sqlite ...
No coverage
/home/phil/git/owncloud/core/lib/composer/phpunit/phpunit/phpunit --configuration phpunit-autotest.xml --log-junit autotest-results-sqlite.xml ../tests/lib/Files/External/StoragesBackendServiceTest.php 
PHPUnit 9.6.19 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.4.33
Configuration: phpunit-autotest.xml

....F.                                                              6 / 6 (100%)

Time: 00:00.023, Memory: 20.00 MB

There was 1 failure:

1) Test\Files\External\StoragesBackendServiceTest::testUserMountingBackends
OCP\Files\External\Backend\Backend::removeVisibility(1) was not expected to be called.

/home/phil/git/owncloud/core/lib/private/Files/External/StoragesBackendService.php:124
/home/phil/git/owncloud/core/tests/lib/Files/External/StoragesBackendServiceTest.php:177

FAILURES!
Tests: 6, Assertions: 25, Failures: 1.
make: *** [Makefile:194: test-php-unit] Error 1
```

I get this unit test fail locally with your branch but not with `master`

I will have a look at what happens differently in the code.

--- a/tests/lib/Files/External/StoragesBackendServiceTest.php
+++ b/tests/lib/Files/External/StoragesBackendServiceTest.php
@@ -152,7 +152,7 @@ class StoragesBackendServiceTest extends \Test\TestCase {
 		$this->config->expects($this->exactly(2))
 			->method('getAppValue')
 			->will($this->returnValueMap([
-				['files_external', 'allow_user_mounting', 'yes', 'yes'],
+				['files_external', 'allow_user_mounting', 'no', 'yes'],
 				['files_external', 'user_mounting_backends', '', 'identifier:\User\Mount\Allowed,identifier_alias']
 			]));
 

